python/GangaDirac/test/Unit/Utilities/TestDiracUtilitiesExternals.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
Yes but this then requires manipulating globals and renaming all of the tests so that they expliclty run after a global has been changed. See the top of the previous file for what has to be so the tests can run without a proxy. Putting them in a separate file makes things a lot clearer and removes the need for some overly complex wrapper or extension to allow some methods to be called with/without a proxy.